sale_blanket_order/wizard/create_sale_orders.py

```diff
@@ -100,7 +100,7 @@ def _default_lines(self):
     def _prepare_so_line_vals(self, line):
         return {
             "product_id": line.product_id.id,
-            "name": line.product_id.name,
+            "name": line.product_id.display_name,
```

Comment:
@jado95 Thanks for your contribution.

In order to be more compliant with standard Odoo behaviour, could you adapt it with this : https://github.com/odoo/odoo/blob/14.0/addons/sale/models/sale.py#L1714 ?

Thanks